Added:

Mononofuu opened the pull request:
- Animal repository
- Animal get by _House_ specification
- Animal get by _Species_ specification

skyfenko commented on persistence/src/main/java/edu/softserve/zoo/persistence/repository/impl/AnimalRepositoryImpl.java:
```diff
@@ -0,0 +1,14 @@
+import org.springframework.stereotype.Repository;
+
+/**
+ * <p>Realisations of all {@link AnimalRepository} specific addons for {@link Animal} entity</p>
```
Interesting thing to note: check the difference between realization and implementation in translator

Mononofuu replied:
Oops. I'll fix it

skyfenko commented on persistence/src/main/java/edu/softserve/zoo/persistence/specification/impl/animal/AnimalGetAllByHouseIdSpecification.java:
```diff
@@ -0,0 +1,30 @@
+ * @author Serhii Alekseichenko
+ */
+public class AnimalGetAllByHouseIdSpecification implements DetachedCriteriaSpecification<Animal> {
+    private static final String HOUSE_ID = House.class.getSimpleName().toLowerCase() + ".id";
```
if this functionality needed around the project, please implement common method for this stuff

Mononofuu committed:
added:
 - Animal repository
 - Animal get by House specification
 - Animal get by Species specification
 - refactor
